pkg/gui/controllers/tags_controller.go

```diff
@@ -74,6 +74,13 @@ func (self *TagsController) GetKeybindings(opts types.KeybindingsOpts) []*types.
 			DisplayOnScreen:   true,
 			OpensMenu:         true,
 		},
+		{
+			Key:             opts.GetKey(opts.Config.Branches.MergeIntoCurrentBranch),
+			Handler:         opts.Guards.OutsideFilterMode(self.withItem(self.merge)),
+			Description:     self.c.Tr.Merge,
+			Tooltip:         self.c.Tr.MergeBranchTooltip,
```

Comment:
We should add a new tooltip called MergeTagTooltip because the current tooltip only mentions branches

Reply:
Got it, I'll work on this when I get some free time along with the other comment.

Reply:
Added this in 91a620ce49d7, let me know if I should squash these down.